cypress/integration/configuration/shippingMethods/channelsInShipping.js

```diff
@@ -18,9 +16,13 @@ import * as shippingUtils from "../../../support/api/utils/shippingUtils";
 import filterTests from "../../../support/filterTests";
 import { getFormattedCurrencyAmount } from "../../../support/formatData/formatCurrencyAmount";
 import { enterHomePageChangeChannelAndReturn } from "../../../support/pages/channelsPage";
+import {
+  enterAndSelectShippings,
+  enterShippingZone
+} from "../../../support/pages/shippingZones";
 
 filterTests({ definedTags: ["all"] }, () => {
-  describe("Channels in shippingMethod", () => {
+  describe("As a staff user I want have different shipping method prices for each channel", () => {
```

Comment:
This test fails during execution, probably changes to UI were made which resulted in error
![image](https://user-images.githubusercontent.com/97674352/153202943-6f4963ac-4ede-410c-b91c-94047ef3a613.png)